frontend/src/components/form/base/ETimePicker.vue

```diff
@@ -57,25 +57,46 @@ export default {
   },
   methods: {
     allowedStep: m => m % 15 === 0,
+    parseTime (value) {
+      if (this.valueFormat === 'x') {
+        return this.$date.utc(value)
+      } else {
+        return this.$date.utc(value, this.valueFormat)
+      }
+    },
+    formatTime (time) {
+      if (this.valueFormat === 'x') {
+        return time.valueOf()
+      } else {
+        return time.format(this.valueFormat)
+      }
+    },
+    setTime (initialDateTime, newDateTime) {
+      return initialDateTime
+        .hour(newDateTime.hour())
+        .minute(newDateTime.minute())
+        .second(newDateTime.second())
+        .millisecond(newDateTime.millisecond())
+    },
     format (val) {
       if (val !== '') {
-        this.dateTime = this.$moment.utc(val, this.valueFormat)
+        this.dateTime = this.parseTime(val)
         return this.dateTime.format('LT')
       }
       return ''
     },
     formatPicker (val) {
       if (val !== '') {
-        return this.$moment.utc(val, this.valueFormat).format(this.$moment.HTML5_FMT.TIME)
+        return this.parseTime(val).format('HH:mm')
       }
       return ''
     },
     parse (val) {
       if (val) {
-        const m = this.$moment.utc(val, 'LT')
-        this.dateTime.hours(m.hours()).minutes(m.minutes()).seconds(m.seconds()).milliseconds(m.milliseconds())
-        if (m.isValid()) {
-          return Promise.resolve(this.dateTime.format(this.valueFormat))
+        const parsedDateTime = this.$date.utc(val, 'LT')
+        if (parsedDateTime.isValid() && parsedDateTime.format('LT') === val) {
+          this.dateTime = this.setTime(this.dateTime, parsedDateTime)
```

Comment:
this.setTime throws an NPE if the TimePicker is initialized without a value. (as in /controls).
TypeError: initialDateTime is null
    setTime ETimePicker.vue:75
    parsePicker ETimePicker.vue:111
    inputPicker BasePicker.vue:173
    VueJS 4
    emitValue VTimePicker.ts:198
    onInput VTimePicker.ts:241
    VueJS 4
    update VTimePickerClock.ts:234
    setMouseDownValue VTimePickerClock.ts:229
    onDragMove VTimePickerClock.ts:206
    onMouseDown VTimePickerClock.ts:180
    VueJS 

I would expect that setTime sets the state directly and knows the previous state.
Now the user of setTime (which seems to set the dateTime Object), has to check if this.dateTime is null or not.

Either move the null check into setTime and set the time there,
or rename setTime to createDateTimeFrom.

And because the NPE happened now, a test for empty initialization would be nice.